packages/amplify_storage_s3/lib/method_channel_storage_s3.dart

```diff
@@ -31,8 +33,13 @@ class AmplifyStorageS3MethodChannel extends AmplifyStorageS3 {
     try {
       return await _channel.invokeMethod('addPlugin');
     } on PlatformException catch (e) {
-      throw AmplifyException("Storage plugin has already been added, " +
-          "multiple plugins for Storage category are currently not supported.");
+      if (e.code == "AmplifyAlreadyConfiguredException") {
```

Comment:
Not a blocker by any means but should error codes be enumerated? This question broadly applies to the other packages as well.. regardless of dart vs swift vs kt

Reply:
In this particular case, the value we are inspecting is just a simple string, as this is data that has been serialized across the Flutter platform channel.